cpp/throwaways/generate_gnd_association_and_spelling_replacements.sh

```diff
@@ -0,0 +1,93 @@
+# Create krimdok local keyword file from the original output and make misspellings and potential GND references explicit
+#!/bin/bash
+
+
+if [ $# != 4 ]; then
+    echo "usage: $0 A-Z_output A_Z_bereinigt gnd_full.mrc result"
```

Comment:
Please do not mix german and english parameter names, and also use consistent prefixes.
If the first parameter is called `A-Z_output`, the second one should be renamed from `A_Z_bereinigt` to something like `A-Z_cleaned`.

Reply:
This is intentional because the original file names provided by K.D. are named like this and the script only works with these.